common/ivy.xml

```diff
@@ -16,7 +16,8 @@
   <dependency org="org.dom4j" name="dom4j" rev="${dom4j.version}" />
   <dependency org="ical4j" name="ical4j" rev="0.9.16-patched" />
   <dependency org="junit" name="junit" rev="4.8.2" />
-  <dependency org="log4j" name="log4j" rev="1.2.16" />
+  <dependency org="org.apache.logging.log4j" name="log4j-core" rev="2.17.1" />
```

Comment:
there is a way to specify this version at a central location instead of hardcoding in each and every xml file, check dom4j.version in same file

Reply:
> there is a way to specify this version at a central location instead of hardcoding in each and every xml file, check dom4j.version in same file

Done.

Reply:
Add it here too please:
https://github.com/Zimbra/zm-zcs/blob/develop/ivysettings.xml

Looks like jetty version is missing too.

Reply:
added jetty version in zm-zcs